CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/deps)
 # To configure which apps/operators to build
 include(HoloHubConfigHelpers)
 
+# Set install directory
+set(CMAKE_INSTALL_PREFIX "${CMAKE_SOURCE_DIR}/install")
```

Comment:
Let's use `CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT` to allow the user to override our default setting:
```suggestion
# Set install directory
if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
  set_property(CACHE CMAKE_INSTALL_PREFIX PROPERTY VALUE "${CMAKE_SOURCE_DIR}/install")
endif()
```
Ref: https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT.html#variable:CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT

Reply:
Cool! Thanks!